containers/kubernetes/how-to/upgrade-kubernetes-version.mdx

```diff
@@ -32,11 +37,124 @@ categories:
     <Message type="important">
       Be careful when upgrading the Kubernetes versions of your node pools, as it may lead to data loss on data stored locally on any node.
     </Message>
- 5. Click **Upgrade**. 
+ 5. Click **Upgrade**.
     <Message type="note">
       It is not possible to downgrade your Kubernetes version once it has been upgraded. 
     </Message>
 
+## Upgrading a Kapsule cluster to the next minor version using the CLI
+
+<Message type="requirement">
+    - You have a working [CLI](https://github.com/scaleway/scaleway-cli) with your credentials set up
+    - You are willing to upgrade your Kubernetes Kapsule cluster to the latest k8s version available on the Kapsule API.
+</Message>
+
+
+To begin, it is essential to verify that the most recent version of Kapsule adequately supports your workload. We maintain a compatibility matrix for various components, as your current cluster might use components deprecated or unavailable in the latest version.
+For further details, please consult our [version policy](https://www.scaleway.com/en/docs/containers/kubernetes/reference-content/version-support-policy/).
+
+We recommend you read the Kubernetes [changelog](https://kubernetes.io/releases/) to stay informed on the latest version upgrade.
+
+### Checking which components must be changed
+
+Run the following command into your terminal to retrieve a list of the components that need to be changed.
+
+```
+scw k8s version list
+```
+
+You should get an output similar to the following, providing a list of all relevant components:
+```
+1.25.9   [cilium calico kilo]                [none]                 [containerd]
+1.24.13  [cilium calico weave flannel kilo]  [none]                 [containerd crio]
+1.23.17  [cilium calico weave flannel kilo]  [none nginx traefik2]  [containerd crio docker]
+```
+
+### Ingresses
```

Comment:
Is it right that this is plural?

Reply:
I think so - but I will change it to ### Ingress controllers maybe?